### DISCUSSION TITLE
LineのメッセージングAPIで認証を聞かれます。ipadのLineで登録できる術はあります？

### DISCUSSION THREAD

**Discussion by SuzumitsuTs1103** [1ft-seabass/ma-hackathon-qa-and-knowledge-202210#2] [Q&A] [answered]
LineのメッセージングAPIで認証を聞かれます。ipadのLineで登録できる術はあります？ 買うしかない？？
@1ft-seabass

**Comment by 1ft-seabass** [comment, marked as answer]
「認証を聞かれます。」のところ、詳しく聞かせてください。こちら LINE BOT の作成中に起きたこと、ということでよろしかったでしょうか～？

あるいは、LINE BOT でチャネルアクセストークンとチャンネルシークレットを設定しているが、認証を聞かれるということでしょうか～。

**Reply by SuzumitsuTs1103** [reply]
おー！！ありがとうございます。
①https://developers.line.biz/ja/services/messaging-api/
　で「今すぐ始めよう」押下
②Lineアカウントでログイン押下
③Lineのログインで、メアドとPWを登録してログイン押下
③Lineに４桁認証番号が飛びますが、ipadのみの場合、認証番号が届かない・・・

**Reply by 1ft-seabass** [reply]
ああ！なるほど！メアドとPWのログイン、ひょっとすると、そのあたりの操作がややこしくなりやすいので、下部の QR コードを LINE アプリで読み取り後に出てくる認証番号で対応されたほうが分かりやすいかもしれません！

![image](https://user-images.githubusercontent.com/2148840/195088952-73c4e014-f70a-4cb7-bceb-159608f9e589.png)

**Reply by 1ft-seabass** [reply]
QR コード読み取りの操作で迷われたらこちらで。
![image](https://user-images.githubusercontent.com/2148840/195089474-48f42b44-59e3-40b6-84a4-039bc7733e69.png)

**Reply by SuzumitsuTs1103** [reply]
ありがとうございます。試してみます。！！

**Reply by 1ft-seabass** [reply]
@SuzumitsuTs1103 Slack の案内で正式運用は、10/17 くらいからとアナウンスされましたが、このスレッドはお気軽に試してくださいね。それで解決したら、解決クローズしてみましょうー。

**Reply by SuzumitsuTs1103** [reply]
ありがとうございます。上記試みましたが、ダメでした。
さらにググると、スマホ所持が前提のように読めました。ipadのみやPcのみ所持のユーザは認証が飛ばない仕様のようです。
なんか、抜け道はないか模索しましたが、スマホを買うしかないようです。

**Reply by 1ft-seabass** [reply]
ああーなるほど！そういうことでしたか。ちょっと所持の状況うまく分かってなかったです。

であれば、抜け道なさそうなので、スマホ所持ですかねー。

**Reply by SuzumitsuTs1103** [reply]
closeします。ありがとうございました

**Reply by 1ft-seabass** [reply]
クローズありがとうございましたー。ディスカッション本格稼働したら、またよろしくお願いします！
